lib/api.js

```diff
@@ -152,6 +152,23 @@ class Madge {
 		return graph.image(this.obj(), this.circular(), imagePath, this.config);
```

Comment:
```suggestion
if (this.circular) {
    return this.circularImage(imagePath);
} else {
    return graph.image(this.obj(), this.circular(), imagePath, this.config);
}
```
We can add the check here to generate image for --circular dependencies option.
This is better as it solves the linting issue of complexity and too many statements which is causing the build to fail